generators/server/templates/src/main/java/package/domain/User.java.ejs

```diff
@@ -370,7 +370,7 @@ public class <%= user.persistClass %><% if (databaseTypeSql || databaseTypeMongo
     @Relationship("HAS_AUTHORITY")
 <%_ } _%>
 <%_ if (databaseTypeSql && !reactive) { _%>
-    @ManyToMany
+    @ManyToMany(fetch = FetchType.EAGER)
```

Comment:
Should we really make this EAGER?

Reply:
I made it eager because otherwise I faced a org.hibernate.LazyInitializationException during Asynchronous indexing. That was the easiest fix and the impact sounded rather small to me, as it is just a single other table being joined.

Reply:
https://stackoverflow.com/questions/17278385/spring-async-generates-lazyinitializationexceptions I'll check if putting the `Transactional` onto the @Async method will help :-)